src/Exceptions/HttpImateapot.php

```diff
@@ -2,10 +2,10 @@
 
 namespace Tnapf\Router\Exceptions;
 
-class HttpImateapot extends HttpException {
+class HttpImateapot extends HttpException
```

Comment:
HttpImATeapot or HttpTeapot, capitalization is bad here

Reply:
Probably due to how it is in my json file